surfactant/cmd/generate.py

```diff
@@ -380,16 +406,10 @@ def sbom(
                             except Exception as e:
                                 raise RuntimeError(f"Unable to process: {filepath}") from e
 
-                            if file_is_symlink and entry.installPrefix:
-                                # Track the symlink, but don't add to list of entries
-                                # as it'll be processed later anyways
-                                if sw_parent.sha256 not in file_symlinks:
-                                    file_symlinks[sw_parent.sha256] = []
-                                file_symlinks[sw_parent.sha256].append(install_filepath)
-                            else:
-                                entries.append(sw_parent)
-                                for sw in sw_children:
-                                    entries.append(sw)
+                            print(f"===Adding {sw_parent.installPath}")
```

Comment:
This print statement should use the logger instead

Reply:
Oh, I forgot to remove that print.